_episodes/markdown.md

```diff
@@ -76,205 +72,79 @@ we can use it as a text editor and investigate selected Markdown features.
+```
+# group-website
+Repo for learning how to make websites with Jekyll pages
 
+## Learning Markdown
 
-> ## Add a Sub Heading and Emphasised Text with Line Breaks
->
-> Try to reproduce the source code of the following view.
->
-> <img src="../fig/group_website_exercise_add_a_sub_heading.png" alt="Preview tab of README.md with additional lines inlcuding sub heading and emphasised text with line breaks" width=400>
->
->
-> > ## Solution
-> >
-> >     # group-website
-> >     Repo for learning how to make websites with Jekyll pages
-> >
-> >     Some **bold** font
-> >
-> >     ## Learning Markdown
-> >
-> >     Vanilla text may contain *italic* and **bold words**.
-> >
-> >     This paragraph is separated from the previous one by a blank line.
-> >     Line breaks
-> >     are caused by two trailing spaces at the end a line.
-> >
-> {: .solution }
-{: .challenge }
+Vanilla text may contain *italic* and **bold words**.
 
-Displaying items in lists, sometimes increases readability.
-You can create lists by preceding lines with `-` or a bunch of other `-`-like characters.
+This paragraph is separated from the previous one by a blank line.
+Line breaks  
+are caused by two trailing spaces at the end of a line.
 
-> ## List today's learning goals
->
-> Give it a try and write a short list of topics you want to learn today.
->
-> > ## Solution
-> >
-> >     - Write Markdown
-> >     - Host my first website on GitHub
-> >     - Learn about Jekyll
-> >     - Think about how my work group could benefit from what I have learned.
-> >
-> {: .solution }
-{: .challenge }
+[Carpentries Webpage](carpentries.org)
 
-If you would like to nest lists, i.e. specify a sub list as an item of another list, use indentation.
-Numbered enumerations are helpful for displaying sequences and the numbers may help to address individual points.
+### Carpentries Lesson Programs:
+- Software Carpentry
+- Data Carpentry
+- Library Carpentry
+```
 
-> ## Nested Lists and Enumerations
->
-> Try to reproduce the source code of the following view.
->
-> <img src="../fig/exercise_nested_lists_enumerations.png" alt="Preview tab nested lists and enumerations" width=600>
->
-> > ## Solution
-> >
-> >     - A list of items
-> >     - can be
-> >       - deeply
-> >         - nested
-> >     - returning to the top-level like this.
-> >
-> >     2. Enumerations can start at any number but
-> >     3. continue to increase in steps of one
-> >     5. even if the numbering does not. This updates the rendered numbering to avoid gaps when individual items are deleted.
+You can then click the preview tab again to see how the formatting renders.
+
+<img src="../fig/markdown_preview_formatting.png" alt="Preview of the formatting added to the Readme" width=600>
+
+To keep this addition to our `README.md` we need to commit these changes to save them.
+Scroll down to the bottom of the page, add a commit message if you wish, and then commit to the `main branch`.
```

Comment:
Maybe we could fit the screenshot below referring to `branch` main instead of `branch main`

```suggestion
Scroll down to the bottom of the page, add a commit message if you wish, and then commit to the `main` branch.
```